lib/preload.js

```diff
@@ -356,14 +392,18 @@ preload.run = (resin, docker, options) => {
   })
   .then((token) => {
     options.apiToken = token
-    return getContainerSize(options.apiHost, options.apiToken, options.registryHost, options.imageRepo)
+    return Promise.all([
+      getContainerSize(options.apiHost, options.apiToken, options.registryHost, options.imageRepo),
```

Comment:
Looks like this function could benefit from just taking `options` as an argument, rather than taking 4 arguments.

Reply:
I'd rather not change it in this pull request as it is not related.

Reply:
Yeah, makes sense!